### ISSUE TITLE
Prepare audit log with appropriate permissions

### ISSUE THREAD

**Issue by randomvariable** [kubernetes-sigs/image-builder#191]
To enable auditing in Kubernetes, ideally would have this go to `/var/log/kubernetes/audit/audit*.log`

This would be done by setting the `extraVolumes` options on kubeadm, however the default permissions are not suitable for audit logs.

By prepping the file with appropriate permissions beforehand, Kubernetes API Server keeps the audit log secure.

The following should be sufficient:

``` yaml
- name: directory.logs.kubernetes
  file:
    path: /var/log/kubernetes
    state: directory
- name: kubernetes.log.audit
  file:
    path: /var/log/kubernetes/audit.log
    state: touch
    mode: "0700"
    owner: root
    group: root
```

**Comment by fyang-pivotal** [comment]
@randomvariable Out of curiosity, why "0700" permission, granting executable permission as well for the root user? Would "0600" permission be sufficient?

**Comment by fyangd** [comment]
@randomvariable This issue has been resolved in PR https://github.com/kubernetes-sigs/image-builder/pull/213.   If you enable audit logging and point it to the precreated `/var/log/kubernetes/audit.log` file, you should observe the appropriate permission.  

Feel free to try it out and close and the issue.

**Comment by codenrhoden** [comment]
/remove-lifecycle stale

@randomvariable Can you confirm that this issue has been resolved?

**Comment by randomvariable** [comment]
I think this is only resolved for Photon, and maybe affected by #314 when I just went to test on Photon.

**Comment by randomvariable** [comment]
/close

After we sorted out the clean up of /var/log, this problem has gone away.
